modules/react/select/stories/Select.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Select} from '@workday/canvas-kit-react/select';
+
+import {Alert} from './examples/Alert';
+import {AlertStandalone} from './examples/AlertStandalone';
+import {Default} from './examples/Default';
+import {Disabled} from './examples/Disabled';
+import {DisabledOption} from './examples/DisabledOption';
+import {Error} from './examples/Error';
+import {ErrorStandalone} from './examples/ErrorStandalone';
+import {Grow} from './examples/Grow';
+import {GrowStandalone} from './examples/GrowStandalone';
+import {LabelPosition} from './examples/LabelPosition';
+import {RefForwarding} from './examples/RefForwarding';
+import {Required} from './examples/Required';
+import {Standalone} from './examples/Standalone';
+
+<Meta title="Components/Inputs/Select/React" component={Select} />
+
+# Canvas Kit Select
+
+Select Drop-downs allow users to choose one option from a list of items in a Drop-down Menu.
```

Comment:
Since the majority of people are starting to use the Select component in preview, it would be nice to link to it here (and maybe in the readme).

Reply:
Done. I've added it to [the top of the Select docs](https://5d854c26ba934e0020f5e98a-duglscczpo.chromatic.com/?path=/docs/components-inputs-select-react--default-story) -- let me know if you have any suggestions on wording.